packages/office-ui-fabric-react/src/components/Panel/Panel.tsx

```diff
@@ -181,7 +181,7 @@ export class Panel extends BaseComponent<IPanelProps, IPanelState> implements IP
                 { onRenderNavigation(this.props, this._onRenderNavigation) }
               </div>
               <div className={ css('ms-Panel-contentInner', styles.contentInner) } >
-                { onRenderHeader(this.props, this._onRenderHeader) }
+                { onRenderHeader(this.props, this._onRenderHeader, headerTextId) }
```

Comment:
nit: it should match popup, with headerText && headerTextId. An optional change though.